response_operations_ui/views/collection_exercise.py

```diff
@@ -975,13 +988,25 @@ def get_seft_collection_instrument(short_name, period):
 @login_required
 def post_seft_collection_instrument(short_name, period):
     verify_permission("surveys.edit", session)
-    if "unselect-ci" in request.form:
-        return _unselect_seft_collection_instrument(short_name, period)
+    if "delete-ci" in request.form:
```

Comment:
Confirmed with Gareth, that unlinking a SEFT is now invalid and should be just delete to avoid orphaned data